GitUI/CommandsDialogs/FormCreateBranch.Designer.cs

```diff
@@ -158,8 +158,6 @@ private void InitializeComponent()
             // 
             // tableLayoutPanel1
             // 
-            this.tableLayoutPanel1.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left)
-            | System.Windows.Forms.AnchorStyles.Right)));
```

Comment:
Shouldn't the FormCreateBranch ctor set the anchors a for the other two, should it?